cli/Cargo.toml

```
@@ -0,0 +1,14 @@
+[package]
+name = "cli"
+version = "0.1.0"
+authors = ["Alexandra Østermark <alex.cramt@gmail.com>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+maplit = "1.0.2"
+rayon = "1.3.1"
+lazy_static = "1.4.0"
+log = "0.4.11"
+clap = "2.33.1"
```

I think these can be a little less specific (I know IntelliJ tends to fill in with the most specific possible) because of how semantic versioning works. e.g. 
```suggestion
clap = "2.33"
```